config.tutorial.yaml

```diff
@@ -3,21 +3,13 @@
 # SPDX-License-Identifier: CC0-1.0
 
 version: 1.0.0
+
```

Comment:
Which purpose does have this additional empty line?

Reply:
Hi @ekatef, same as in the config.default.yaml. Just me being a bit pedantic hehe. I will remove it as in config.default.yaml. Thanks for your feedback :)